libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -41,6 +44,16 @@ export class BookSearchComponent {
 
   addBookToReadingList(book: Book) {
     this.store.dispatch(addToReadingList({ book }));
+    // show snack bar message with undo action
+    const snackBarRef = this.snackBar.open('Added "' + book.title + '" to reading list.', 'Undo', { duration: 10000});
+    snackBarRef
+      .onAction()
+      .pipe(take(1))
```

Comment:
You don't need to do this, like many http requests this should complete when it emits.

Reply:
Updated.